environment.yml

```diff
@@ -98,7 +98,8 @@ dependencies:
 
   - pyqt>=5.9.2  # pandas.read_clipboard
   - pytables>=3.4.3  # pandas.read_hdf, DataFrame.to_hdf
-  - s3fs  # pandas.read_csv... when using 's3://...' path
+  - s3fs>=0.4.0  # pandas.read_csv... when using 's3://...' path
```

Comment:
gcsfs?

Reply:
Was not included here previously. There are many FS backends we *could* include, in theory, but I don't imagine anyone wants to write tests for them right now.

Reply:
sure but we are testing it already, its consistent to add it here